### PR TITLE
Add export_hostgroup_fact  snippet

### PR DESCRIPTION
**Pull request by amirfefer**
Template part of http://projects.theforeman.org/issues/6952

### CODE REVIEW

File: snippets/export_hostgroup_fact.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<%#
+kind: snippet
+name: export_hostgroup_fact
+description: this snippet export foreman hostgroup for hostgroup provisioning, hence when the machine registered with Foreman, it will associate with this hostgroup. 
+%>
+<% if @provisioning_type == 'hostgroup' -%>
+export FACTER_foreman_hostgroup=<%= @host.title %>
```
Comment (dLobatog) [inline snippets/export_hostgroup_fact.erb:7]: `@host.hostgroup.title if @host.hostgroup.present?`

**Review by dLobatog — CHANGES_REQUESTED** [review]

**Comment by dLobatog** [comment]
@amirfefer Shouldn't this be added to all finish templates? I'm not sure how to add this to userdata though...

**Comment by mmoll** [comment]
@amirfefer what's the status here?

**Comment by mmoll** [comment]
@amirfefer ping?

**Comment by tbrisker** [comment]
Thanks @amirfefer !

**Pull request closed without merging**